charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -294,3 +294,19 @@ aac-manage-case-assignment:
           - idam-noc-approver-username
           - idam-noc-approver-password
           - notify-mca-api-key-secret
+
+ccd-case-document-am-api:
+  enabled: true
+  java:
+    releaseNameOverride: ${SERVICE_NAME}-cdam
+    imagePullPolicy: Always
+    image: hmctspublic.azurecr.io/ccd/case-document-am-api:latest
+    ingressHost: ccd-case-document-am-api-${SERVICE_FQDN}
+    environment:
+      IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
+      IDAM_OIDC_URL: https://idam-web-public.aat.platform.hmcts.net
+      OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
+      S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+      DM_STORE_BASE_URL: http://dm-store-aat.service.core-compute-aat.internal
```

Comment:
One question (most probably a stupid one :) ) ... how do we know what the value for ${SERVICE_NAME} is? From above I guess that it's "aat.service.core-compute-aat.internal"?

If so I guess we could reuse it here?

Reply:
ah think it's `fpl-case-service-pr-${PR_NUM}`, so it keeps the pod names separate per PR